spk/sonarr/Makefile

```
@@ -35,6 +32,17 @@ POST_STRIP_TARGET = sonarr_extra_install
 
 include ../../mk/spksrc.spk.mk
 
+# use alternate TMPDIR as /tmp might be too small and not accessible on DSM >= 7.1.
+USE_ALTERNATE_TMPDIR = 1
+
+# replace previous legacy package
+INSTALL_REPLACE_PACKAGES = nzbdrone
```

We could nail the package replacement to specific versions. With the following change, there will be no need to handle existing sonarr v2 versions.
```suggestion
INSTALL_REPLACE_PACKAGES = nzbdrone>=20210329
```